File: app/src/main/java/huce/fit/appreadstories/story/list/online/StoryPresenter.java
package huce.fit.appreadstories.story.list.online;

import huce.fit.appreadstories.story.list.BaseStoryListPresenter;

public interface StoryPresenter extends BaseStoryListPresenter {

    void totalPage();
    void getStory(int page);
    void loadNextPage();
    boolean isLoading();
    boolean isLastPage();
    void swipeData();
}

File: app/src/main/java/huce/fit/appreadstories/story/list/online/StoryPresenter.kt
package huce.fit.appreadstories.story.list.online

import huce.fit.appreadstories.story.list.BaseStoryListPresenter

interface StoryPresenter : BaseStoryListPresenter {
    fun totalPage()
    fun getStory(page: Int)
    fun loadNextPage()
    fun isLoading(): Boolean
    fun isLastPage(): Boolean
    fun swipeData()
}
